registry/storage-drivers/swift.md

```diff
@@ -10,231 +10,31 @@ storage.
-      no
-    </td>
-    <td>
-      Your Openstack domain ID for Identity v3 API. You can either use <code>domain</code> or <code>domainid</code>.
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>trustid</code>
-    </td>
-    <td>
-      no
-    </td>
-    <td>
-      Your Openstack trust ID for Identity v3 API.
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>insecureskipverify</code>
-    </td>
-    <td>
-      no
-    </td>
-    <td>
-      <code>true</code> to skip TLS verification, <code>false</code> by default.
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>chunksize</code>
-    </td>
-    <td>
-      no
-    </td>
-    <td>
-      Size of the data segments for the Swift Dynamic Large Objects. This value should be a number (defaults to 5M).
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>prefix</code>
-    </td>
-    <td>
-      no
-    </td>
-    <td>
-      This is a prefix that is applied to all Swift keys to allow you to segment data in your container if necessary. Defaults to the empty string which is the container's root.
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>secretkey</code>
-    </td>
-    <td>
-      no
-    </td>
-    <td>
-      The secret key used to generate temporary URLs.
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>accesskey</code>
-    </td>
-    <td>
-      no
-    </td>
-    <td>
-      The access key to generate temporary URLs. It is used by HP Cloud Object Storage in addition to the <code>secretkey</code> parameter.
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>authversion</code>
-    </td>
-    <td>
-      no
-    </td>
-    <td>
-      Specify the OpenStack Auth's version, for example <code>3</code>. By default the driver autodetects the auth's version from the AuthURL.
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>endpointtype</code>
-    </td>
-    <td>
-      no
-    </td>
-    <td>
-      The endpoint type used when connecting to swift. Possible values are <code>public</code>, <code>internal</code>, and <code>admin</code>. Default is <code>public</code>.
-    </td>
-  </tr>
-</table>
+| Parameter     | Required | Description                                                                                                                                                                                                                                                         |
+|:--------------|:---------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `authurl`  |  yes  | URL for obtaining an auth token. https://storage.myprovider.com/v2.0 or https://storage.myprovider.com/v3/auth |
+| `username`  |  yes  |	Your Openstack user name. |
+| `password`  |  yes | Your Openstack password. |
+| `region`  | no   |The Openstack region in which your container exists. |
```

Comment:
Does this require a space? (haven't tried building this PR as I was waiting for some other builds to complete 😅)

```suggestion
| `region`  | no   | The Openstack region in which your container exists. |
```

(some more like that below)

From a readability perspective, it'd be great if we could align the columns in the Markdown source, but that can be a pain to do (unless your editor allows automatically formatting it)

Reply:
Yes, that needs a space. Thanks for catching that. I'll take a look for other occurrences.

Reply:
I> Does this require a space? (haven't tried building this PR as I was waiting for some other builds to complete 😅)

You can check the preview here - https://5e567880b5e20800072d1e32--angry-visvesvaraya-c62c8e.netlify.com/registry/storage-drivers/s3/